modules/hcp-virt-create-hc-cli.adoc

```diff
@@ -75,4 +75,4 @@ NAMESPACE   NAME      VERSION   KUBECONFIG                 PROGRESS    AVAILABLE
 clusters    example   4.x.0     example-admin-kubeconfig   Completed   True        False         The hosted control plane is available
 ----
 +
-Replace `4.x.0` with the supported {product-title} version that you want to use.
\ No newline at end of file
+Replace `<4.x.0>` with the supported {product-title} version that you want to use.
```

Comment:
On line 75, be sure to place `4.x.0` in angle brackets to match the annotation text.